Library/ViewModels/CommentRepliesViewModel.swift

```diff
@@ -248,18 +251,25 @@ public final class CommentRepliesViewModel: CommentRepliesViewModelType,
       .skipNil()
       .takeWhen(self.dataSourceLoadedProperty.signal)
 
-    // TODO: Call blocking GraphQL mutation
-    self.userBlocked = self.blockUserProperty.signal.map { true }
+    let blockUserEvent = self.blockUserProperty.signal
+      .map(BlockUserInput.init(blockUserId:))
+      .switchMap { input in
+        AppEnvironment.current.apiService
+          .blockUser(input: input)
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+          .materialize()
+      }
 
-    self.userBlocked.observeValues { didBlock in
-      guard didBlock == true else { return }
-      NotificationCenter.default.post(.init(name: .ksr_blockedUser))
-    }
+    self.didBlockUser = blockUserEvent.values().ignoreValues()
+      .map { _ in NotificationCenter.default.post(.init(name: .ksr_blockedUser)) }
+
+    // TODO: Display proper error messaging from the backend
+    self.didBlockUserError = blockUserEvent.errors().ignoreValues()
```

Comment:
@santiDotIO asked about this, but do we even want to bother with server-driven error messages here? I can think of some that might be nice (this user has already been blocked, for example) but I'm assuming errors here are such a small edge case. Might not be worth the work to plumb them in and translate them.